projects/admin/src/app/acquisition/components/order/order-email-form/order-email-form.component.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2023 RERO
+ * Copyright (C) 2023 UCLouvain
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { AcqOrderApiService } from '@app/admin/acquisition/api/acq-order-api.service';
+import { IAcqOrder, IAcqOrderPreviewResponse } from '@app/admin/acquisition/classes/order';
+import { Notification } from '@app/admin/classes/notification';
+import { ITypeEmail } from '@app/admin/shared/preview-email/component/preview-email/preview-email.component';
```

Comment:
why not `IRecipient` (seems more relevant) ? Personnaly, I had place this interface into a specific file (@app/admin/shared/interface) instead into a component

Reply:
This interface is part of the module. I'm not going to move it to admin/shared